From acab47a7c442dec48d97b00ee7c88b7f002a82a8 Mon Sep 17 00:00:00 2001
From: Akira Moroo <retrage01@gmail.com>
Date: Sat, 12 Sep 2020 20:08:31 +0900
Subject: [PATCH] Add 9pfsPkg to supported packages for CI

Signed-off-by: Akira Moroo <retrage01@gmail.com>
---
 .pytool/CISettings.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/.pytool/CISettings.py b/.pytool/CISettings.py
index 0c2dd0181a..79c6d98d5c 100644
--- a/.pytool/CISettings.py
+++ b/.pytool/CISettings.py
@@ -55,7 +55,8 @@ class Settings(CiBuildSettingsManager, UpdateSettingsManager, SetupSettingsManag
                 "FatPkg",
                 "CryptoPkg",
                 "UnitTestFrameworkPkg",
-                "OvmfPkg"
+                "OvmfPkg",
+                "9pfsPkg"
                 )
 
     def GetArchitecturesSupported(self):
-- 
2.17.1

